pyproject.toml

```
@@ -28,7 +28,7 @@ dependencies = [
     "httpx",
     "importlib_metadata>=4.6; python_version<'3.10'",
     "packaging",
-    "panel>=1.3,<1.4",
+    "panel>=1.3.6",  # GitHub issue #240
```

1. Let's keep the upper pin for now. The chat components are under active development and had major changes between 1.2 and 1.3. I don't want Ragna to break for users if something like that happens for 1.4 and this gets automatically installed.
2. No need for the comment. `git blame` will show the PR that touched this and in turn also link to the issue.

```suggestion
    "panel>=1.3.6,<1.4",
```